ipatests/test_integration/test_cert.py

```diff
@@ -127,6 +128,84 @@ def test_ipa_getcert_san_aci(self):
         ipaddrs = ext.value.get_values_for_type(x509.IPAddress)
         assert ipaddrs == [ipaddress.ip_address(self.clients[0].ip)]
 
+    def test_getcert_list_profile(self):
+        """
+        Test that getcert list command displays the profile
+        for the requests
+        """
+        for requests in os.listdir(paths.CERTMONGER_REQUESTS_DIR):
+            result = self.master.run_command(
+                ["getcert", "list", "-i", requests], raiseonerr=False
+            )
+            assert "profile: caSubsystemCert" in result.stdout_text
+            assert "profile: caSignedLogCert" in result.stdout_text
+            assert "profile: caOCSPCert" in result.stdout_text
+            assert "profile: caSubsystemCert" in result.stdout_text
+            assert "profile: caCACert" in result.stdout_text
+            assert "profile: caServerCert" in result.stdout_text
+            assert "profile: caIPAserviceCert" in result.stdout_text
+            assert "profile: caIPAserviceCert" in result.stdout_text
+            assert "profile: KDCs_PKINIT_Certs" in result.stdout_text
+
+    def test_getcert_list_profile_using_subca(self):
+        """
+        Test that getcert list command displays the profile
+        for the request with a SUBCA configured on IPA server
+        """
+        sub_name = "CN=SUBCA"
+        tasks.kinit_admin(self.master)
+        self.master.run_command(
+            ["ipa", "ca-add", "mysubca", "--subject=" + sub_name]
```

Comment:
It would be nice to use `"%s" % (var)` instead of concatenation.

Reply:
In fact you can just do [..., "--subject", sub_name]

Reply:
@mrizwan93 have made the change.